terraform/modules/iam_user/s3_opensearch/outputs.tf

```diff
@@ -0,0 +1,19 @@
+output "username" {
+  value = var.username
+}
+
+output "access_key_id_prev" {
+  value = ""
+}
+
+output "secret_access_key_prev" {
+  value = ""
+}
+
+output "access_key_id_curr" {
+  value = aws_iam_access_key.iam_access_key_v1.id
+}
+
+output "secret_access_key_curr" {
+  value = aws_iam_access_key.iam_access_key_v1.secret
+}
```

Comment:
These need to be outputs of the main stack so we can read them, like so:

https://github.com/cloud-gov/terraform-provision/blob/ef28b7234d43d70bd086c48d4b921632b5dab268/terraform/stacks/main/outputs.tf#L690-L697

I'd call the output `logs_opensearch_s3_user_access_key_id_curr`, ec